pkg/operator/bootstrapteardown/bootstrap_teardown_controller.go

```diff
@@ -254,6 +254,9 @@ func (c *BootstrapTeardownController) isKASConfigValid(kasOperator *operatorv1.K
 	// appropriate storageConfig
 	for _, revision := range uniqueRevisions {
 		configMapNameWithRevision := fmt.Sprintf("%s-%d", configMapName, revision)
+		if revision == 0 {
+			configMapNameWithRevision = configMapName
```

Comment:
I think I prefer the "notfound" error I would expect.  This shouldn't do work until the revision is present, right?  You could choose to skip at a higher level.

Reply:
modified it to skip if revision is 0, check for other node's revision, if all are 0, it would return false at the bottom, PTAL @deads2k